### ISSUE TITLE
Tag a release?

### ISSUE THREAD

**Issue by cstjean** [MikeInnes/MacroTools.jl#111]
Please? It would be nice for us not to depend on #master in our release.

**Comment by MikeInnes** [comment]
You have commit, so I think you could just go ahead and tag – let me know if there's any problems or concerns about that though.

**Comment by cstjean** [comment]
I'd forgotten about that. It's been merged into METADATA.
